### PR TITLE
Add version into log

### PR DESCRIPTION
**Pull request by jichenjc**
**What this PR does / why we need it**:

add versions of git into manager's log

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

**Comment by jichenjc** [comment]
recheck

**Comment by jichenjc** [comment]
recheck

File: version/version.go [hunk 1]
```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```
Comment (hidekazuna) [inline version/version.go:2]: now 2020, not 2018.

**Review by hidekazuna — CHANGES_REQUESTED** [review]

File: version/version.go (hunk 1 above)
Comment (jichenjc) [reply version/version.go]: updated, thanks

**Pull request merged**